seismostats/analysis/tests/test_estimate_beta.py

```diff
@@ -94,129 +96,107 @@ def test_estimate_b(
         assert w[-1].category == UserWarning
 
 
+def magnitudes(b: float):
+    df_mags = pd.read_csv(
+        'seismostats/analysis/tests/data/simulated_magnitudes.csv')
+    if b == 0.5:
+        mags = df_mags["b_value = 0.5"].values
+    elif b == 1:
+        mags = df_mags["b_value = 1"].values
+    elif b == 1.5:
+        mags = df_mags["b_value = 1.5"].values
+    return mags
+
+
 @pytest.mark.parametrize(
-    "n,b,mc,delta_m,b_parameter,precision",
+    "b, mags, mc, delta_m, precision, b_parameter",
     [
-        (1000000, 1.2, 3, 0, "b_value", 0.005),
-        (1000000, np.log(10), 3, 0.1, "beta", 0.01),
+        (1, magnitudes(1), 0, 0.1, 0.002, "b_value"),
```

Comment:
use exact b-values calculated with the function as it is now, assuming it is now correct.